pom.xml

```diff
@@ -112,12 +112,14 @@
 		<benchmarkjar.name>benchmarks</benchmarkjar.name>
 		<javac.target>1.8</javac.target>
 		<slf4j.version>1.7.21</slf4j.version>
-		<log4j2.version>2.6.2</log4j2.version>
+		<log4j2.version>2.1</log4j2.version>
```

Comment:
It seems that I can't shade out the logging from Storm.  I can shade the project's logger.  Or we can keep v 2.1 for now until we do modules and Storm has its own module?..  Maybe it's good to shade the project logger though to remove all potential problems?.. I'm very new to shading though. So I'm entirely open to whatever suggestions.